Schema.js

```diff
@@ -75,6 +78,7 @@ function classNameIsValid(className) {
     className === '_Session' ||
     className === '_SCHEMA' || //TODO: remove this, as _SCHEMA is not a valid class name for storing Parse Objects.
     className === '_Role' ||
+    className === '_GlobalConfig' ||
```

Comment:
Thanks, I'll take a look. It seems to be `schemas.js` uses `req.config.database.collection` to access a collection which eventually ends up with `classNameIsValid` too but I'll dig a bit deeper tonight.